ground/src/test/java/com/google/android/ground/ui/map/MapControllerTest.kt

```diff
@@ -16,18 +16,20 @@
 package com.google.android.ground.ui.map
 
 import android.location.Location
+import app.cash.turbine.test
 import com.google.android.ground.BaseHiltTest
+import com.google.android.ground.coroutines.ApplicationScope
 import com.google.android.ground.model.geometry.Coordinates
 import com.google.android.ground.model.geometry.Point
 import com.google.android.ground.repository.LocationOfInterestRepository
 import com.google.android.ground.repository.MapStateRepository
 import com.google.android.ground.repository.SurveyRepository
 import com.google.android.ground.system.LocationManager
+import com.google.common.truth.Truth.assertThat
```

Comment:
Are you still having trouble with tests? If so, lmk and I can take a look.

Reply:
I've removed the tests for now as the map controller class has been disintegrated and moved into the VM